### ISSUE TITLE
Suggestion about report, share, permissions and mirrors

### ISSUE THREAD

**Issue by dourkan** [Nexus-Mods/web-issues#212]
When those expandable panels weren't available on the previous design, all of the mentioned items were generally manually added by the mod authors at the end of the page, and not the beginning. 

I believe it diverts atenttion from the mod and makes navigating a tiny bit less comfortable, as you have to scroll down to get to the mod (this intensifies when using a phone).

As so, I think they would be better placed at the end of a mod page, and not at the top of it.

**TL;DR:** report, share, permissions/credits and mirrors should be at the bottom of a mod page.
![these](https://user-images.githubusercontent.com/33101691/32017092-1968fcec-b99c-11e7-90d5-00b2ff9e887f.jpg)

**Comment by Thallassa** [comment]
They were at the top of the page before, just harder to find. Making them more obvious is not a bad thing - this is the important information.

**Comment by dourkan** [comment]
![these ones](https://user-images.githubusercontent.com/33101691/32020584-0d2c56a8-b9a7-11e7-8faf-3ca1c60dca6b.jpg)
You mean this. I know, most mod authors used to put credits and permissions at the end of the page (as you pointed out, no one saw nor clicked those), it was just an idea I had.
